packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/health-monitor/create/component.js

```diff
@@ -2,6 +2,13 @@ import template from './template.html';
 import controller from './controller';
 
 export default {
+  bindings: {
+    projectId: '<',
+    region: '<',
+    poolId: '<',
```

Comment:
poolId is a bit redundant, given you have the pool

Reply:
I get it from pool routing resolve where it is already redundant :)